kubevirtci

```diff
@@ -130,7 +130,7 @@ function kubevirtci::up() {
 cert-manager:
   url: "https://github.com/cert-manager/cert-manager/releases/latest/cert-manager.yaml"
 EOF
-	$CLUSTERCTL_PATH init -v 4 --config=${_default_bin_path}/clusterctl_config.yaml
```

Comment:
this will likely pass CI, but i don't think this is what we want. Doesn't using the `--infrastructure kubevirt` flag cause the latest capk release to be installed? In this script, we want the install to come from the local build. see the `sync` logic in kubevirtci.